### PR TITLE
Create signin and signout component in sbc-common-components

### PR DESCRIPTION
**Pull request by nitheesh-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/2657

* created components for signin and signout
* moved keycloak service from sbc-auth to common components

### CODE REVIEW

File: vue/sbc-common-components/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "sbc-common-components",
-  "version": "2.0.53",
+  "version": "2.1.0",
```
Comment (sutherlanda) [inline vue/sbc-common-components/package.json]: Nice 👍

File: vue/sbc-common-components/src/services/keycloak.services.ts [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+  async logout (redirectUrl: string) {
+    let token = ConfigHelper.getFromSession(SessionStorageKeys.KeyCloakToken)
+    if (token) {
+      this.kc = Keycloak(ConfigHelper.getKeycloakConfigUrl())
+      let kcOptions :KeycloakInitOptions = {
+        onLoad: 'login-required',
+        checkLoginIframe: false,
+        timeSkew: 0,
+        token: sessionStorage.getItem('KEYCLOAK_TOKEN'),
```
Comment (sutherlanda) [inline vue/sbc-common-components/src/services/keycloak.services.ts]: Can we use the config helper to retrieve these, just so it's consistent with the rest of the library?  Should be `ConfigHelper.getFromSession(SessionStorageKeys.KeyCloakToken)`

File: vue/sbc-common-components/tests/unit/logout.spec.ts [hunk 3]
```diff
@@ -4,29 +4,36 @@ import { mount, createLocalVue } from '@vue/test-utils'
...
 
 Vue.use(Vuetify)
 Vue.use(VueRouter)
 
-jest.mock('axios', () => ({
-  post: jest.fn(() =>
-    Promise.resolve({ data: { access_token: 'abcd', refresh_token: 'efgh', registries_trace_id: '12345abcde' } })
-  )
-}))
+// jest.mock('axios', () => ({
```
Comment (sutherlanda) [inline vue/sbc-common-components/tests/unit/logout.spec.ts]: Let's just remove this.

File: vue/sbc-common-components/src/services/keycloak.services.ts (hunk 2 above)
Comment (nitheesh-aot) [reply vue/sbc-common-components/src/services/keycloak.services.ts]: sure, will do
Thanks @sutherlanda

Comment (nitheesh-aot) [reply vue/sbc-common-components/src/services/keycloak.services.ts]: done

**Pull request merged**